frontend/src/pages/dashboard/question/QuestionView.tsx

```diff
@@ -1,79 +1,121 @@
 import React, { useEffect } from "react";
-import "./Question.scss";
-import Dropdown from "../../../components/dropdown/Dropdown";
-import Skeleton from "react-loading-skeleton";
 import 'react-loading-skeleton/dist/skeleton.css';
 import { useNavigate, useParams } from "react-router-dom";
-import SplitSection from "../../../components/section/SplitSection";
-import TextEditor from "../../../components/texteditor/TextEditor";
-import Button from "../../../components/button/Button";
-import { Answer, Question } from "../../../def/Question";
-import { formatDate } from "../../../def/converter";
+import { AnswerDef, QuestionDef } from "../../../def/QuestionDef";
 import { useAlert } from "react-alert";
 import { axiosError } from "../../../def/axios-error";
-import NoContent from "../../../components/NoContent";
-import QuestionStats from "./QuestionStats";
-import QuestionAnswerSkeleton from "./QuestionAnswerSkeleton";
-import QuestionAnswer from "../../../components/questionpreview/QuestionAnswer";
-import Avatar from "../../../components/avatar/Avatar";
+import QuestionAnswer from "../../../components/question/QuestionAnswer";
+import Dropdown from "../../../components/dropdown/Dropdown";
+import Button from "../../../components/button/Button";
+import 'react-responsive-modal/styles.css';
+import QuestionTitle from "../../../components/question/QuestionTitle";
+import Question from "../../../components/question/Question";
+import { QuestionSectionTitle } from "../../../components/question/QuestionSectionTitle";
+import { QuestionPaginationNext, QuestionPaginationPrev } from "../../../components/question/QuestionPagination";
+import QuestionDivider from "../../../components/question/QuestionDivider";
+import QuestionAnswerEditor from "../../../components/question/QuestionAnswerEditor";
+import ActiveProfileModal from "../../../components/ActiveProfileModal";
+import { Session } from "@ory/client";
+
+const ANSWERS_PAGE_SIZE = 5;
```

Comment:
Do you have a config, and explain what this constant is about, what does `ANSWERS_PAGE_SIZE` mean?

Reply:
It is the size of a page of answers.